### PR TITLE
bugfix/CDAP-20100: add tooltip for composite trigger configuration

### PR DESCRIPTION
**Pull request by de-lan**
Add tooltip for composite trigger configuration

## Description
Summary of changes

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [x] General Improvement
- [ ] Cherry Pick

## Links
Jira: 
[CDAP-20022](https://cdap.atlassian.net/browse/CDAP-20022)
[CDAP-20100](https://cdap.atlassian.net/browse/CDAP-20100)

## Screenshots
image
image

### CODE REVIEW

File: app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx [hunk 1]
```diff
@@ -55,6 +55,9 @@ import {
 import PayloadConfigModal from 'components/PipelineTriggers/PayloadConfigModal';
 import PipelineCompositeTriggerRow from './PipelineCompositeTriggerRow';
 
+const colors = require('styles/colors.scss');
+const toolTipBackground = colors.blue07;
```
Comment (seanfeld) [inline app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx]: Can you get this some way that doesn't use colors.scss?

File: app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx [hunk 2]
```diff
@@ -338,16 +355,20 @@ const PipelineListCompositeTabView = ({
...
-            color="primary"
-            disabled={state.isNameInvalid || triggersGroupToAdd.length === 0}
-            variant="contained"
-            onClick={() => addGroupTriggerClick()}
-            data-cy="enable-group-trigger-btn"
-            data-testid="enable-group-trigger-btn"
-          >
-            {T.translate(`${PREFIX}.addNewTrigger`)}
-          </EnableGroupTriggerButton>
+          <CustomTooltip arrow title={state.triggerNameError || emptyTriggerErrorMsg}>
```
Comment (seanfeld) [inline app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx:355]: Are you sure this should use a tooltip? usually we try not to use tooltips for stuff that that has intractability?

Comment (sumengwang) [reply app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx:355]: learned a new word today

File: app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx (hunk 1 above)
Comment (de-lan) [reply app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx]: updated, thanks!

File: app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx (hunk 2 above)
Comment (de-lan) [reply app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx:355]: what's the advice here? I've set the error message in the trigger name input, but the configuration page could be too long if there are many upstream pipelines and users will ignore it. We need a proper way to guide users why the enable button is disabled

**Review by sumengwang — APPROVED** [review]

File: app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx (hunk 2 above)
Comment (seanfeld) [reply app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx:355]: use richtooltip

Comment (seanfeld) [reply app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx:355]: i used it for a similar thing in the sidepanel

Comment (seanfeld) [reply app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx:355]: its not that a popup menu is bad per se - its using the `tooltip` component to accomplish it.

Comment (seanfeld) [reply app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx:355]: @sumengwang it should've been https://en.wiktionary.org/wiki/interactability but the dang autocorrect don't like me

Comment (seanfeld) [reply app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx:355]: nvm i read this wrong - i thought the button was part of the tooltip content 😞

**Review by seanfeld — APPROVED** [review]

**Pull request merged**